hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.test.tsx

```diff
@@ -1,12 +1,21 @@
 import React from "react";
 
-import { render, screen } from "testing/utils";
+import { render, screen, waitFor } from "testing/utils";
 
-import { mockTeamMultiple } from "testing/mockData";
+import { mockTeam, mockTeamMultiple } from "testing/mockData";
 import TeamDetail, { PageParams } from "pages/TeamDetail/TeamDetail";
 import { RouteComponentProps } from "react-router-dom";
+import { get } from "api/api";
+import { AxiosResponse } from "axios";
 
-test("renders without crashing", () => {
+jest.mock("api/api", () => ({
+    ...jest.requireActual("api/api"),
+    get: jest.fn(),
+}));
+
+const mockedGet = get as jest.MockedFunction<typeof get>;
+
```

Comment:
Could you wrap both these tests in a `describe("<TeamDetail />")` block

Reply:
done